lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3856,6 +3858,118 @@ def test_show_student_extensions(self):
 
 
 @attr('shard_1')
+class TestCourseIssuedCertificatesData(SharedModuleStoreTestCase):
+    """
+    Test data dumps for issued certificates.
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(TestCourseIssuedCertificatesData, cls).setUpClass()
+        cls.course = CourseFactory.create()
+
+    def setUp(self):
+        super(TestCourseIssuedCertificatesData, self).setUp()
+        self.instructor = InstructorFactory(course_key=self.course.id)
+        self.client.login(username=self.instructor.username, password='test')
+
+    def generate_certificate(self, course_id, mode, status):
+        """
+        Generate test certificate
+        """
+        test_user = UserFactory()
+        GeneratedCertificateFactory.create(
+            user=test_user,
+            course_id=course_id,
+            mode=mode,
+            status=status
+        )
+
+    def test_certificates_features_against_status(self):
+        """
+        Test certificates with status 'downloadable' should be in the response.
+        """
+        url = reverse('get_issued_certificates_features', kwargs={'course_id': self.course.id.to_deprecated_string()})
```

Comment:
We should not use `to_deprecated_string()` instead use `unicode(self.course.id)`